packages/template-retail-react-app/app/components/product-tile/promo-callout.mock.js

```diff
@@ -0,0 +1,1129 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export const productSearch = {
+    // a hit from productSearch call
+    rollSleeveBlouse: {
+        currency: 'GBP',
+        hitType: 'master',
```

Comment:
Do you want to remove any data that is not necessary for the promo call out test to reduce the file size? or you can reuse any data from  app/mocks/product-search-hit-data.js?

Reply:
Nah, we don't need to worry about the weight of the test files :) They will not be included in the bundle, right?

I didn't want to worry about consolidating the mocks too. What if I have a particular product that does not exist in  app/mocks/product-search-hit-data.js ?